source/fe/mapping_q1.cc

```diff
@@ -1391,6 +1391,7 @@ MappingQ1<dim,spacedim>::transform (
                   }
 
               }
+      return;
```

Comment:
Darn, the fact that we missed this `return` statement in the past means that nothing in the testsuite ever executes the block of code above it :-(